### DISCUSSION TITLE
Is it worth publishing a fork of react-loadable?

### DISCUSSION THREAD

**Discussion by iMobs** [vercel/next.js#10715] [Help]
I noticed react-loadable has been effectively [copied](https://github.com/zeit/next.js/blob/master/packages/next/next-server/lib/loadable.js) and updated with new features. The last time an official version of react-loadable was published was over 2 years ago and it will eventually be unusable as it contains deprecated component lifecycle methods. Since the current maintainer of react-loadable doesn't seem interested in making any improvements to the library and does not have any open channels for discussing fixes for these issues does anyone else think that a fork should be created and namespaced as `@next/react-loadable`?

**Comment by timneutkens** [comment]
We're not planning to release next/dynamic as it'll eventually be updated to work in a different way (backed by React.lazy when it supports pre-rendering). It would also introduce unneeded maintenance burden as we recommend using Next.js for building React applications and it comes built-in with dynamic loading.
